src/index.js

```diff
@@ -14,25 +13,31 @@ document.querySelector('#form-section').addEventListener('submit', (e) => {
 
   // Validation
   if (name === '' || points === '') {
-    // UI.showAlert('Please fill in all fields', 'danger', 1500);
+    UI.showAlert('Please fill in all fields', 'danger', 1500);
   } else {
+    // fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
+    //   method: 'POST',
+    //   headers: {
+    //     'Content-Type': 'application/json',
+    //   },
+    //   body: JSON.stringify({
+    //     name: 'Duel of Fates',
+    //   }),
+    // })
+    //   .then((res) => res.json())
+    //   .then((data) => console.log(data.result));
     // Start a new Score
-    const score = new Score(name, points);
-
-    // Add Score to UI
-    UI.addScoresToList(score);
-    // UI.showAlert('Task Added', 'success', 1500);
-
-    // Add Score to LocalStorage
-    Storage.addScore(score);
-
+    API.sendData(name, points);
```

Comment:
- [ ] I kindly suggest that you ensure that your scores list change only when the refresh button is clicked and not the submit button because this is a project requirement.